src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql

```diff
@@ -3,6 +3,8 @@
 CREATE SCHEMA cstore;
 SET search_path TO cstore;
 
+CREATE SEQUENCE cstore_storageid_seq NO CYCLE;
```

Comment:
Can we set the sequence to start at a value higher than any OID -- say, 10000000000 -- to avoid confusion?

Reply:
done